utils/consts.py

```diff
@@ -1,3 +1,4 @@
 PYTHON_IMAGE = "registry.access.redhat.com/ubi9/python-311:latest"
 TOOLBOX_IMAGE = "registry.access.redhat.com/ubi9/toolbox"
 OC_IMAGE = "registry.redhat.io/openshift4/ose-cli"
+EVAL_IMAGE = "quay.io/rcook/vllm_ilab:vllm"
```

Comment:
vllm is broken since cuda lib is not in LD_LIBRARY_PATH, I'm building a new image and will push shortly.

Reply:
I think we're good, it's late for you! I'm building now, too - I needed to update the list of CUDA versions ot have `;` inbetween each version, that was causing the build to fail. This works:
`ENV TORCH_CUDA_ARCH_LIST="5.0;8.0;8.6;8.7;8.9;9.0+PTX"`

Reply:
🤔 this was not failing for me

Reply:
weird - please check this latest version I pushed - I'll build it & run through the pipeline & if green, we'll merge, sg?